ui/app/components/clients/activity.ts

```diff
@@ -12,21 +12,24 @@ import { parseAPITimestamp } from 'core/utils/date-formatters';
 import { calculateAverage } from 'vault/utils/chart-helpers';
 
 import type ClientsActivityModel from 'vault/models/clients/activity';
+import ClientsConfigModel from 'vault/models/clients/config';
+import type ClientsVersionHistoryModel from 'vault/models/clients/version-history';
 import type {
   ClientActivityNewClients,
   ClientActivityMonthly,
   ClientActivityResourceByKey,
 } from 'vault/models/clients/activity';
-import type ClientsVersionHistoryModel from 'vault/models/clients/version-history';
 
 interface Args {
   activity: ClientsActivityModel;
+  config: ClientsConfigModel;
```

Comment:
Is the config model used anywhere here? If not we should remove and any extending component could define it as an Arg if needed.